.travis.yml

```diff
@@ -1,31 +0,0 @@
-os: linux
-language: node_js
-node_js:
-- '14'
-addons:
-  chrome: stable
-before_install: "source scripts/helpers.sh"
-install: yarn install
-before_script: "yarn build"
-script:
-  - "./scripts/travis.sh"
-dist: xenial
-env:
-  global:
-  - NOKOGIRI_USE_SYSTEM_LIBRARIES=true
-  - SAUCE_BROWSER_NAME="chrome"
-  - SAUCE_BROWSER_VERSION=""
-  - SAUCE_PLATFORM="Windows 10"
-  - SAUCE_APPIUM_VERSION=""
-  - SAUCE_DEVICE_NAME=""
-  - SAUCE_DEVICE_ORIENTATION=""
-  - SAUCE_PLATFORM_VERSION=""
-  - SAUCE_PLATFORM_NAME=""
-  - secure: RW99TW7YvGgWt/P8fQQziPbxNITaJMO/4q2iCX72u/PtTu2dkeaCqUiRUi7qyV8jIIk7U4oruxJRFQpTaCIX5+JWZsSwLskoTsspOI9aTzyPdhVNckM1vpA5VfwstdyQTkX7zzKPLdvXK0xj0ZzQ+ACH/SobQNYKiAMvW7FVkIc=
-  - secure: gFSORYQLSBNIK8peGgUOhd0Yj6GzE18HA0GLrmp0PMtobe003poalkNgLsPy2W2QFZbG3bCjbqFMmCbUh5uRnX8eCy/whe+lwXeQsv5uh0EIGWgxVxhk79dCx5cakERusoj2BUgJJ432R7nRXjEW5v2AK9QAOeCG2Dvwdb8WSes=
-notifications:
-  slack:
-    secure: FbvUST5p63eiqJa4dxArVFYzTittyKzSt0+StBEaJwvLtn/ZiW4N2bJovtccThyIMGny0NJz34rh1jlVXUXOzWYWVTgr/fK4vWhN8mJWK17gg5aIfJr9oJ029k/cZB6luJzY6J8Ld/5SZ7Xsi2SE6Dbnntdx9e6gwSs3f8YqKF8qsPeb4Jw07ax6AR+MgSc3vyjZZeR8g6FjxLTzMepSsbfGs7vrmJO9znW+lhf0mYJaHwlaVLt9mMlgfeciCa9HImqVHID597XvcKZpeXq3OL6NCwE1dQcPjB5/xR4Gl3K9I2bOn9gHf6tZf9di0HD8lMnQ71pc45JYOzs3UTQwdQNNjEyG/ZF0YT7Mok0dcMSDUDCXYWpgTohrz5LzXNKyOYAS3FXIfxQwfegFC065h4/Ex1pSVQCB+7/v/qBBkBZuyLhUfATDvVq22zduK3Q+X4dEwzxvNfCwpOvYe4xPdFUVcNUVFCa9D5VZnbUdYZiF5k1daVL7iRTTT17fc4ssfposFdkh+IX332Cb6y6ytkRBchTW64mk0qQ0haLMfDygn2uzfX0orr2M+S06pQ8oIlYkaffoi2UB1C2jHWFxB73/tbT6ivW0NWj22m/7U7y2Enx/qOOy3LL/SQNhaVOy+UIzWrY7wm+2AlNbs1O5pZ88VJQZZKzDkXx7jb/R7hs=
-  on_pull_requests: false
-  on_success: never
-  on_failure: always
```

Comment:
I have never seen notifications from travis on slack for dev.okta

Reply:
We do receive broken Travis alerts via #eng-devex-alerts ...not sure this is it.

Reply:
Will add a follow up to figure out sending failures through slack